lizard_connector/query.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+import urllib.parse
+
+from lizard_connector import jsdatetime
+
+
+class LizardApiImproperQueryError(Exception):
+    pass
+
+
+class QueryDictionary(dict):
+
+    def update(self, E=None, *queries, **f):
```

Comment:
De `f` zegt niet veel.

En waarom is de E met een hoofdletter?

Zeker omdat er in de docstring nogal wat met éénletterige afkortingen (waaronder F-met-hoofdletter) wordt gegoocheld.

Die docstring: zodra je in de docstring met codevoorbeelden uit gaat leggen wat je aan het doen bent... dan moet je wat aan je naamgeving doen. Want ik kan niet lezen wat er hier gebeurt.